README.md

```diff
@@ -1,17 +1,96 @@
-# FemAlert
+# FemAlert 👩🏽‍💗
+
+## ❗ Live Link ❗
+
+Here is the live link to our demo of the dashboard:
+
+[----Live link to dashboard!----](meesterproef.netlify.app/)
+<br><br>
 
 _by [Sharon Veldman](https://github.com/SharonV33), [Johan van Beusekom](https://github.com/johancvb) & [Veerle Prins](https://github.com/veerleprins)_
 
-## Description
+---
+
+## 📝 Description
+
+53% of all women in The Netherlands has experienced harassment in some way. <br>
+Nowadays, unwanted and inappropriate behaviour towards victims keeps getting worse. Especially in clubs and festivals, people are more often the victim of harassment in any possible way. Sexual harassment and intimidation, aggression, racism and/or discrimination towards victims are mostly the main problems in todays nightlife.
+
+That's where FemAlert comes in handy. FemAlert is an initiative, established by Rosanne Verbeek, to protect people (mostly women) from unwanted situations while partying or going out to clubs.
+FemAlert is a platform where people can report any kind unacceptable behavior in an accessible way, without immediately taking it to the police. FemAlert also gives an option to get in touch with the victim to provide them help and support.
```

Comment:
*any kind of unacceptable
*FemAlert also lets club or bar owners get in touch with victims to help them and offer support